src/pages/Categories.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
 import '../styles/Categories.css';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/Categories/Category';
 
-const Categories = () => (
-  <button type="button" className="update">
-    Check Status
-  </button>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  const handleClick = () => {
+    dispatch(checkStatus());
+  };
+  return (
+    <div className="categories">
+      <button type="button" className="update" onClick={handleClick}>
+        Check Status
+      </button>
+    </div>
+  );
+};
```

Comment:
The page should display `Under construction` when the button is clicked, however, nothing happens you click on the button. Kindly update to reflect this requirement. :wink: